src/PointwiseFunctions/GeneralRelativity/Transform.hpp

```diff
@@ -16,24 +16,26 @@ class DataVector;
 /// Holds functions related to transforming between frames.
 namespace transform {
 //@{
-/// \ingroup GeneralRelativityGroup
-/// Transforms tensor to different frame.
-///
-/// The formula for transforming \f$T_{ij}\f$ is
-/// \f{align}
-///   T_{\bar{i}\bar{j}} &= T_{ij}
-///      \frac{\partial x^i}{\partial x^{\bar{i}}}
-///      \frac{\partial x^j}{\partial x^{\bar{j}}}
-/// \f}
-/// where \f$x^i\f$ are the source coordinates and
-/// \f$\bar{x}^i\f$ are the destination coordinates.
-///
-/// Note that `Jacobian<DestFrame,SrcFrame>` is the same type as
-/// `InverseJacobian<SrcFrame,DestFrame>` and represents
-/// \f$dx^i/dx^{\bar{j}}\f$.
-///
-/// In principle `to_different_frame` can be extended/generalized to
-/// other tensor types if needed.
+/*!
+ * \ingroup GeneralRelativityGroup
+ * Transforms tensor to different frame.
+ *
+ * The formula for transforming \f$T_{ij}\f$ is
+ * \f{align}
+ *   T_{\bar{i}\bar{j}} &= T_{ij}
```

Comment:
[optional] Barred, etc., i and j often look better without the dots (i.e., `\imath` and `\jmath`).